src/Domain/Package/PackageManager.php

```diff
@@ -284,6 +300,12 @@ private function containsValidVersion($data) : bool {
    * Adds a package to the list of packages.
    */
   private function addPackage(array $datum, FixturePathHandler $fixture_path_handler, string $package_name): void {
+    $orca_sut_dir = $this->env->get('ORCA_SUT_DIR');
+    if (empty($datum['url']) && !is_null($orca_sut_dir)) {
+      $orca_sut_dir = $this->env->get('ORCA_SUT_DIR');
+      $package_name_parts = explode('/', $orca_sut_dir);
+      $datum['url'] = "../" . end($package_name_parts);
+    }
```

Comment:
This method is called on in `::initializePackages()` above on every package in `packages.yml`, which I believe means this would change the `url` of _all_ packages to the SUT dir. Obviously, that's not right--this needs to be done somewhere else that targets just the SUT. When you move it, why don't you add some test coverage that would've caught this.